### PR TITLE
Extension of validations

### PR DESCRIPTION
**Pull request by noudcorten**
# What has been changed?

* When value in config exceeds the configured max or min, it is set to the configured default value
* Removed unnecessary function for fee validation, since max and min are now set in specification.json

### CODE REVIEW

File: resources/specification.json [hunk 1]
```diff
@@ -111,8 +112,8 @@
     "description": "fee as percentage, FI: \"0.25%\" ",
     "type": "float",
     "default": 0.25,
-    "min": 0,
-    "max": 100
+    "min": 0.1,
+    "max": 0.5
```
Comment (sheridavandenbent) [inline resources/specification.json:116]: Why do we put a hard limit at 0.5%? Are there no exchanges ever that have a 1% fee, for example?

**Comment by sheridavandenbent** [comment]
The branch name isn't what the pull request is about - It's not a big thing but it bugs me a bit 😅  Can you next time make sure the branch name is up-to-date with what you're working on? That also makes it easier to collaborate as it's more clear what you're working on!

File: resources/specification.json (hunk 1 above)
Comment (noudcorten) [reply resources/specification.json:116]: Good question. It was defined this way in the earlier version, so I just repeated those numbers. We could just remove the limits?

Comment (sheridavandenbent) [reply resources/specification.json:116]: Fair enough - let's leave it for now since it's already the current limit.

**Review by sheridavandenbent — APPROVED** [review]

**Pull request merged**